tests/run/string-extractor.scala

```diff
@@ -39,9 +41,9 @@ object Test {
   }
 
   def g(x: Any): String = x match {
-    case TripleBippy(3 :: Nil, 3.0, 'b', chars : _*)       => "1: " + chars
-    case TripleBippy(5 :: Nil, 5.0, 'b' | 'B', chars : _*) => "2: " + chars
-    case TripleBippy(_, _, chars : _*)                     => "3: " + chars
+    case TripleBippy(3 :: Nil, 3.0,  Bippy('b', chars : _*))      => "1: " + chars
```

Comment:
Why do we need this change?

Reply:
Currently Dotty does not support `TripleBippy` as it's written before. I agree this is not a proper fix. We need to decide whether or not to suppor the Scala2 behavior.

Reply:
Agreed. The spec is also inconsistent as it claims to support a super set of scalac patterns.

Reply:
Note that this test was changed inhttps://github.com/scala/scala/commit/308ae2d3967cf885f2aa5c5c2489b79abb865d42#diff-f761b9d2bf9ec2f17bc00f552fc07314 when Scala 2.13 changed how name-based unapplySeq works: https://github.com/scala/scala/pull/7068

Reply:
Thanks for the link @smarter 👍